### PR TITLE
add a log formatter

### PR DESCRIPTION
**Pull request by balda-rdx**

### CODE REVIEW

File: node-runner-cli/tests/fixtures/docker-config.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ core_node:
   enable_transaction: 'true'
   java_opts: --enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m
     -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts
-    -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
+    -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.log_util.log4j.core.async.AsyncLoggerContextSelector
```
Comment (shambupujar) [inline node-runner-cli/tests/fixtures/docker-config.yaml:22]: I think this is not write as the string replaced is a java option and nothing to do with nodecli logging

File: node-runner-cli/tests/fixtures/docker-compose.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ services:
     environment:
       JAVA_OPTS: --enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m
         -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts
-        -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
+        -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.log_util.log4j.core.async.AsyncLoggerContextSelector
```
Comment (shambupujar) [inline node-runner-cli/tests/fixtures/docker-compose.yaml:9]: Same here as mentioned in below comment . This is java class

File: node-runner-cli/tests/fixtures/docker-config.yaml (hunk 1 above)
Comment (balda-rdx) [reply node-runner-cli/tests/fixtures/docker-config.yaml:22]: ah! good catch! this was the IDE automatically renaming stuff. reverting

File: node-runner-cli/tests/fixtures/docker-compose.yaml (hunk 2 above)
Comment (balda-rdx) [reply node-runner-cli/tests/fixtures/docker-compose.yaml:9]: yep. reverting

**Pull request merged**